player_characters/Theren.md

```diff
@@ -0,0 +1,21 @@
+<!--Needed for Minimo Scribbles-->
+import Admonition from '@theme/Admonition';
+
+# Theren
+
+![Theren](/img/players/Theren.jpg#gh-dark-mode-only)
+![Theren](/img/players/Theren_2.jpg#gh-light-mode-only)
+
+## Background
+He is the brother of Reysatra and very overprotective towards her.
+
+<Admonition type="caution" icon="🐀" title="">
```

Comment:
Why not just use the `:::` block syntax?

Reply:
Maybe this should be Info instead of Caution? Can you include a screenshot of what this looks like in the PR description?

Reply:
<img width="1005" alt="Screenshot 2023-09-25 at 4 40 46 PM" src="https://github.com/LemurianAdventureGuild/wiki/assets/47254813/baec3528-d5f0-4e9a-80e1-b3da85155045">

Reply:
I could do info. I thought the yellow worked better though.

Reply:
You can supposedly import an .svg into the icon but I'm still trying to figure out how to do that. Minimo's scribbles are going to appear in a few select places in the Archives 😈